File: Valghalla.Internal.Web/src/features/participant/models/participant-personal-record.ts
export interface ParticipantPersonalRecord {
  firstName?: string;
  lastName?: string;
  streetAddress?: string;
  postalCode?: string;
  city?: string;
  municipalityCode?: string;
  municipalityName?: string;
  countryCode?: string;
  countryName?: string;
  coAddress?: string;
  age: string;
  birthdate: Date;
  deceased: boolean;
  disenfranchised: boolean;
  exemptDigitalPost: boolean;
  lastValidationDate?: Date;
}

File: Valghalla.Application/CPR/ParticipantPersonalRecord.cs
namespace Valghalla.Application.CPR
{
    public record ParticipantPersonalRecord
    {
        public string? FirstName { get; init; }
        public string? LastName { get; init; }
        public string? StreetAddress { get; init; }
        public string? PostalCode { get; init; }
        public string? City { get; init; }
        public string? MunicipalityCode { get; set; }
        public string? MunicipalityName { get; set; }
        public string? CountryCode { get; set; }
        public string? CountryName { get; set; }
        public string? CoAddress { get; init; }
        public int Age { get; init; }
        public DateTime Birthdate { get; init; }
        public bool Deceased { get; init; }
        public bool Disenfranchised { get; init; }
        public bool ExemptDigitalPost { get; init; }
        public DateTime? LastValidationDate { get; init; }
    }
}
